include.js

```diff
@@ -5932,6 +5932,13 @@ let cards = [
     author: 'ShueiYang',
     githubLink: 'https://github.com/ShueiYang'
   }
```

Comment:
Hi @AlanGonzalez2022 , you may want to check to see if you omitted a comma around here. You've got a great work with the learning race. Well done!